build-system/tasks/karma.conf.js

```diff
@@ -156,8 +156,10 @@ module.exports = {
   client: {
     mocha: {
       reporter: 'html',
-      // Longer timeout on Travis; fail quickly at local.
-      timeout: process.env.TRAVIS ? 10000 : 2000,
+      // Allow tests to run for up to 5 seconds locally and on Travis.
+      // TODO(rsimha-amp): Reduce local run timeout to 2s after large tests are
```

Comment:
There might be some legit "large" unit test. Could we increase their timeout individually instead? Having this large timeout makes development very painful if tests are timing out.